<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetEditReferenceImageData</class>
 <widget class="QWidget" name="WidgetEditReferenceImageData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QFrame" name="frameDashBoardReference">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 40px;
     max-height: 40px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}


</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_25">
            <item row="0" column="0">
             <widget class="QLabel" name="labelDashBoardReference">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Camera  Reference Data</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBoxShowRuler">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelShowMeasureRuler">
             <property name="text">
              <string>Show Ruler</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonEditCameraROI">
           <property name="text">
            <string>Center Windows</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButtonLoadReferenceFromDisk">
           <property name="text">
            <string>Load Reference Imge</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButtonCheckNewReferenceFromCamera">
           <property name="text">
            <string>Get New Image </string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pushButtonAddNewRect">
           <property name="text">
            <string>Add  Window</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pushButtonDeleteSelectedRect">
           <property name="text">
            <string>Delete  Window</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButtonSaveAllReferenceData">
           <property name="text">
            <string>Save Reference Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="ReferenceImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>264</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>264</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {

 background:rgb(249,247,248);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_20">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_21"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0" colspan="2">
          <widget class="QFrame" name="frameDashBoardTitleChangeSelectedWindow">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>264</width>
             <height>42</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
     background:rgb(249,247,248);
     min-height: 40px;
     max-height: 40px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QLabel" name="labelDashBoardChangeSelectedWindow">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Selected Window</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="frameDashBoardROIMeasureXPosition">
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>82</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
     background:rgb(249,247,248);
    
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
 border: px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_16">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelDashBoardROISelectedWindowXPosition">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>X Position</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxSelectedWindowXPosition">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string> pix</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFrame" name="frameDashBoardROIMeasureWidth">
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>82</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
     background:rgb(249,247,248);
     
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_17">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelDashBoardSelectedWindowWidth">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxSelectedWindowWidth">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string> pix</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frameDashBoardROIMeasureYPosition">
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>82</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
     background:rgb(249,247,248);
    
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_18">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelDashBoardSelectedWindowYPosition">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Y Position</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxSelectedWindowYPosition">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string> pix</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QFrame" name="frameDashBoardROIMeasureHeight">
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>82</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
     background:rgb(249,247,248);
     
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  border: 0px solid rgb(0,255,0);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 20px;
  max-height: 20px;
}

QDoubleSpinBox{
background:rgb(255,255,255);
  color: rgb(39,51,66);
  font-size: 16px;
  font-weight: bold;
  font-family: Siemens Sans;
  min-height: 32px;
  max-height: 32px;
}

QDoubleSpinBox::focus{
 border: 2px solid rgb(0,95,135);
}

</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_19">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelDashBoardSelectedWindowHeight">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxSelectedWindowHeight">
              <property name="font">
               <font>
                <family>Siemens Sans</family>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QDoubleSpinBox{
     background:rgb(255,255,255);
     color:rgb(39,51,66);
}</string>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="suffix">
               <string> pix</string>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>1.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
